tests/test_webpack.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test Webpack module."""
+
+import pytest
+from flask_webpackext import WebpackBundle
+from pywebpack import UnsupportedExtensionError
+
+from invenio_assets.webpack import UniqueJinjaManifestEntry, \
+    UniqueJinjaManifestLoader, WebpackThemeBundle
+
+
+def test_webpack_theme_bundle(app):
+    """Test WebpackThemeBundle."""
+    themes = {
+        'bootstrap3': dict(
+            entry={
+                'theme': './theme-bootstrap3.css',
+            }
+        ),
+        'semanticui': dict(
```

Comment:
```suggestion
        'semantic-ui': dict(
```
just to align with existing? same below